### PR TITLE
Update PullApprove to version 2

### PR DESCRIPTION
**Pull request by Kostronor**
Review please @Minecolonies/maintainers

We now better represent our workflow.

Once Github releases a Pull Request Review api I will be there and integrate that :D

### CODE REVIEW

File: .pullapprove.yml [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
-    - maintainers
+version: 2
+groups:
+  maintainers:
+    required: 3
+    author_approval:
+      auto: true
+    approve_by_comment: 
+      enabled: true
+      approve_regex: '^(Approved|:shipit:|:\+1:|LGTM)'
```
Comment (cltnschlosser) [inline .pullapprove.yml]: Do we want to require the approval at the beginning on the comment ( ^ ), didn't have it before

**Comment by Raycoms** [comment]
Always only accepted the approval at the beginning, as far as I know.

2016-09-23 10:00 GMT-03:00 Colton Schlosser <notifications@github.com>:

File: .pullapprove.yml (hunk 1 above)
Comment (Kostronor) [inline .pullapprove.yml]: I think it will make sense, that way it it intentional and not a random :+1: somewhere in a review.

**Comment by Kostronor** [comment]
@Raycoms it works as intended

**Comment by Raycoms** [comment]
:shipit:

**Comment by cltnschlosser** [comment]
:shipit:

**Pull request merged**